.github/workflows/tests.yaml

```diff
@@ -38,11 +38,11 @@ jobs:
     steps:
       - run: echo "setting versions"
     outputs:
-      # vault-k8s version
-      K8S_VERSIONS: '["1.29.2", "1.28.7", "1.27.11", "1.26.14", "1.25.16"]'
-      VAULT_N: "1.16.1"
-      VAULT_N_1: "1.15.6"
-      VAULT_N_2: "1.14.10"
+      # JSON encoded array of k8s versions.
+      K8S_VERSIONS: '["1.30.0", "1.28.10", "1.27.14", "1.26.14"]'
```

Comment:
https://github.com/kubernetes-sigs/kind/releases

Looks like 1.28.9 is the latest patch for 1.28.x
Similarly, 1.27.13 for 1.27.x.

Somehow, they didn't keep up with [kubernetes](https://kubernetes.io/releases/)?

Reply:
Updated based on https://github.com/kubernetes-sigs/kind/releases/tag/v0.23.0 -

Fixed in 5c2bff2a2f56a583353205ba4ea1ee70a0f2f331